step1/README.md

```diff
@@ -47,24 +49,46 @@ This role will require the following permissions:
 
 Paste the following commands, replacing `{Subnet-Value}` and `{VPC-Value}`.  
 
-You can find these in the AWS console, by navigating to the `VPC` service, and selecting the ID of a public subnet, or one that allows port 80 inbound.  
+You can find these in the AWS console, [by navigating to the VPC service, and selecting the ID of a public subnet.](https://ap-southeast-2.console.aws.amazon.com/vpc/home?region=ap-southeast-2)
+
+*Note:* Subnet is considered public if it has direct access to internet via internet gateway. If you
+are operating within default VPC created by AWS on your behalf, during account creation, all subnets
+will be public by default. VPC ID can also be discovered from subnet's console, in "VPC" column. 
+
+<img alt="public subnet img" src="https://user-images.githubusercontent.com/1170273/142021113-b45706a5-6585-477e-9e70-68ef4b89719b.png" />
+
 
 ``` aws
-aws cloudformation create-stack --stack-name ec2-metadata-ssrf --template-body file://vuln-ec2-template.yaml --parameters ParameterKey="WebServerIngressLocation",ParameterValue="$IP_ADDRESS/32" ParameterKey="Subnet",ParameterValue="{Subnet-Value}" ParameterKey="DeploymentVPC",ParameterValue="{VPC-Value}" --capabilities CAPABILITY_NAMED_IAM
+aws cloudformation create-stack --stack-name ec2-metadata-ssrf \
+    --template-body file://vuln-ec2-template.yaml \
+    --parameters \
+     ParameterKey="WebServerIngressLocation",ParameterValue="$IP_ADDRESS/32" \
+     ParameterKey="Subnet",ParameterValue="{Subnet-Value}" \
+     ParameterKey="DeploymentVPC",ParameterValue="{VPC-Value}" \
+     --capabilities CAPABILITY_NAMED_IAM --region eu-south-1
```

Comment:
Can you remove the eu-south-1 region flag on this one, think most people will likely be using ap-southeast-2